### ISSUE TITLE
[Bug] Mitigation / Reimport failing

### ISSUE THREAD

**Issue by Pionerd** [telekom-mms/trivy-dojo-report-operator#62]
### Description

I have some issues with getting the mitigation and reimport set up correctly. 

Because I wanted to have fixes in the Kubernetes cluster automatically updated, I set `DEFECT_DOJO_CLOSE_OLD_FINDINGS` to true. After that, yesterday, somehow all my findings were suddenly closed, even though report CRDs still existed in the cluster. 

At that moment, I assumed this was a glitch, but to get things resolved, I also set `DEFECT_DOJO_DO_NOT_REACTIVATE` to false, so that I hoped the findings would be opened again.

At this point however, I see the following annotation occurring:
```
    kopf.zalando.org/send_to_dojo: '{"started":"2024-03-14T10:51:22.826299+00:00","delayed":"2024-03-14T11:55:55.838974+00:00","purpose":"create","retries":55,"success":false,"failure":false,"message":"HTTP
      error occurred: 500 Server Error: Internal Server Error for url: https://defectdojo.curacao.pionerds.nl/api/v2/reimport-scan/
      - b''{\"message\":\"Internal server error, check logs for details\"}''. Retrying
      in 60 seconds"}'
```

The defect dojo logs show errors related to duplicate keys:
```
django.db.utils.IntegrityError: duplicate key value violates unique constraint "dojo_test_import_finding_test_import_id_finding_i_e6f366e5_uniq"
DETAIL:  Key (test_import_id, finding_id)=(1668, 2433) already exists.
```

Basically I have two questions/concerns:
- How could everything get set to mitigated in the first place? 
- Why are the findings not correctly reactivated again? 

### Reproduction steps

See above.

### Current Behavior

See above.

### Expected Behavior

See above.

**Comment by rndmh3ro** [comment]
I'm not sure this is related to the operator but a problem with your setup or defectdojo.

We basically just push data to the reimport-API, no additional logic.

There's a similar bug described here: https://github.com/DefectDojo/django-DefectDojo/issues/6217
Maybe there's a race-condition here, too?

I any way, we'd need a proper reproduction-setup to test this.

**Comment by Pionerd** [comment]
Ah, thank you for pointing that issue out. Given the immense number of likes, I will wait for that one to progress firstly.
